examples/llm_and_nlp/hf-dataset-llm-eval.py

```diff
@@ -0,0 +1,59 @@
+from huggingface_hub import InferenceClient
```

Comment:
@shcheklein this test looks flaky: https://github.com/iterative/datachain/actions/runs/11655280717/job/32449776121

Reply:
yep, I'm looking into this ... probably bc we run it in parallel and that's a bit too much for a real e2e test ... I'll if we can run it on a single machine and python